pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -636,7 +660,16 @@ func (asw *actualStateOfWorld) AddPodToVolume(markVolumeOpts operationexecutor.M
 	}
 
 	podObj, podExists := volumeObj.mountedPods[podName]
-	if !podExists {
+
```

Comment:
not your change, but why we use podNmae as key here? Why not podUID? what if two different pods with the same name (old and new)?

Reply:
actually I think we use Pod uid as pod unique name

Reply:
Yes, podName is UID. I'm not sure why it's so confusing.